### ISSUE TITLE
Link for a new repo diff not work

### ISSUE THREAD

**Issue by ZerooCool** [go-gitea/gitea#8777]
I had a problem originally nothing to do with Gitea but with my client FileZilla, the images of my site were no longer searchable on my linux mint.
I uninstalled filezilla and deleted its configuration, then reinstalled filezilla. It's good, my images are searchable.
So I copy the files and images, again, in my repository, to have clean files
I observe then that only some rights are modified, from 644 to 755, and, that very rare files (8) see their end of line to change (CRLF for LF)
so, basically, no new files!
No content changes directly.
I git add.
git commit
push git

If I want to consult the link of the modifications made, I have the following error message:
template: repo/diff/image_diff:14:29: executing “repo/diff/image_diff” at : error calling call: call of nil

Other link when a file is add or delete work good.

( Mon installation : https://wiki.visionduweb.fr/index.php?title=Installer_un_serveur_Git_avec_Gitea )

**Comment by lunny** [comment]
Which version of your gitea instance? I remember there is a PR fixed that.

**Comment by zeripath** [comment]
#8289

**Comment by zeripath** [comment]
In all seriousness we have an issue template for a reason. You have to tell us which version you're using.

**Comment by ZerooCool** [comment]
https://wiki.visionduweb.fr/index.php?title=Installer_un_serveur_Git_avec_Gitea

-->
https://wiki.visionduweb.fr/index.php?title=Installer_un_serveur_Git_avec_Gitea#T.C3.A9l.C3.A9charger_le_binaire

-->
Ma versions master : 1.10.0+dev-359-gbcd4af483 built with GNU Make 4.1, go1.13.1 : bindata, sqlite, sqlite_unlock_notify

**Comment by zeripath** [comment]
Ok, your version is older than the commit that merged #8289 . We've released two 1.10 release candidates and made at least 192 commits since the tagging 1.10-rc1 which #8289 was part of. I suggest you update.

I'm gonna close this as fixed and a duplicate. If, after updating, the problem persists please reopen.

**Comment by ZerooCool** [comment]
Link work with 1.10.0-RC1.

Outside of Gitea's context, I can not understand why a new commit was created for all my files, because of the change in rights CHMOD does it seem to you (FTP uploads and rights concerns) )

No changes were made to the content, but some of the files were added to the repository due to a CHMOD change.

We can not observe the CHMOD of a file, front end, from the web browser, it seems to me?

Therefore, the link of the new repository, even if it works, is not very speaking.

Anyway, the link works and no longer leads to an error.
So it's a good thing for navigating the tool.
Thank you.
